### PR TITLE
Ncl 5167

### PR DESCRIPTION
**Pull request by janinko**
### Checklist:

* [ ] Have you added a note in the [CHANGELOG wiki](https://github.com/project-ncl/pnc/wiki/Changelog) for your change if user-facing?
* [ ] Have you added unit tests for your change?

### CODE REVIEW

**Comment by janinko** [comment]
retest this please

File: dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java [hunk 1]
```diff
@@ -36,17 +35,16 @@
 @ToString(callSuper = true)
 @EqualsAndHashCode(callSuper = true)
 @JsonDeserialize(builder = TargetRepository.Builder.class)
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class TargetRepository extends TargetRepositoryRef {
```
Comment (michalovjan) [inline dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java]: What is the point of now having TargetRepository full DTO?

Comment (janinko) [reply dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java]: None, it's not even used.

Comment (janinko) [reply dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java]: And never have been.

Comment (michalovjan) [reply dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java]: If we removed it, would it cause issues? It's just for clean code.

**Comment by janinko** [comment]
retest this please

File: dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java (hunk 1 above)
Comment (janinko) [reply dto/src/main/java/org/jboss/pnc/dto/TargetRepository.java]: Removed

**Pull request merged**